docs/articles/expensify-classic/integrations/travel-integrations/Trip-Actions.md

```diff
@@ -1,5 +1,47 @@
----
-title: Coming Soon
-description: Coming Soon
----
-## Resource Coming Soon!
+--
+title: Navan Integration
+description: Connecting Navan and Expensify
+--
```

Comment:
I think these need to remain with 3 dashes, otherwise Jekyll doesn't understand you're trying to set the page title

```suggestion
---
title: Navan Integration
description: Connecting Navan and Expensify
---
```